src/ansys/mapdl/core/launcher.py

```diff
@@ -1139,6 +1142,13 @@ def launch_mapdl(
 
               export PYMAPDL_MAPDL_VERSION=22.2
 
+    detect_HPC: bool, optional
+        Whether detect if PyMAPDL is running on an HPC cluster or not. Currently
+        only SLURM clusters are supported. By detaul, it is set to true.
+        This option can be bypassed if the environment variable
+        ``PYMAPDL_ON_SLURM`` is set to "true". For more information visit
+        :ref:`ref_hpc_slurm`.
+
     kwargs : dict, optional
         These keyword arguments are interface specific or for
         development purposes. See Notes for more details.
```

Comment:
```suggestion
        These keyword arguments are interface-specific or for
        development purposes. For more information, see Notes.
```
Where Is "Notes". It seems that this should be a link?

Reply:
It is part of the docstring... meaning that under the `Parameters` section (where we are now), you can find another section called Notes. Unfortunately, I don't think we can link to those headers.